src/dfktools/quests/quest_v1.py

```diff
@@ -5,6 +5,7 @@ class Quest:
     def __init__(self, rpc_address, logger):
         self.rpc_address = rpc_address
         self.logger = logger
+        self.realm = "serendale"
```

Comment:
It might be best to actually exclude this and just do it the way V2 would do it.